Docs/source/install/hpc/adastra.rst

```diff
@@ -0,0 +1,146 @@
+.. _building-adastra:
+
+Adastra (CINES)
+===============
+
+The `Adastra cluster <https://www.cines.fr/calcul/adastra/>`_ is located at CINES (France).
+Each node contains 4 AMD MI250X GPUs, each with 2 Graphics Compute Dies (GCDs) for a total of 8 GCDs per node.
+You can think of the 8 GCDs as 8 separate GPUs, each having 64 GB of high-bandwidth memory (HBM2E).
+
+
+Introduction
+------------
+
+If you are new to this system, **please see the following resources**:
+
+* `Adastra user guide <https://dci.dci-gitlab.cines.fr/webextranet/user_support/>`_
+* Batch system: `Slurm <https://dci.dci-gitlab.cines.fr/webextranet/user_support/index.html?highlight=sbatch#running-jobs>`_
+* `Production directories <https://dci.dci-gitlab.cines.fr/webextranet/data_and_storage/index.html#data-and-storage>`_:
+
+  * ``$SHAREDSCRATCHDIR``: meant for short-term data storage, shared with all members of a project, purged every 30 days (17.6 TB default quota)
+  * ``$SCRATCHDIR``: meant for short-term data storage, single user, purged every 30 days
+  * ``$SHAREDWORKDIR``: meant for mid-term data storage, shared with all members of a project, never purged (4.76 TB default quota)
+  * ``$WORKDIR``: meant for mid-term data storage, single user, never purged
+  * ``$STORE`` : meant for long term storage, single user, never purged, backed up
+  * ``$SHAREDHOMEDIR`` : meant for scripts and tools, shared with all members of a project, never purged, backed up
+  * ``$HOME`` : meant for scripts and tools, single user, never purged, backed up
+
+
+Installation
+------------
+
+Use the following commands to download the WarpX source code and switch to the correct branch:
+
+.. code-block:: bash
+
+   git clone https://github.com/ECP-WarpX/WarpX.git $HOME/src/warpx
+
+We use the following modules and environments on the system (``$HOME/adastra_warpx.profile``).
+
+.. literalinclude:: ../../../../Tools/machines/frontier-olcf/adastra_warpx.profile.example
+   :language: bash
+   :caption: You can copy this file from ``Tools/machines/adastra-cines/adastra_warpx.profile.example``.
+
+We recommend to store the above lines in a file, such as ``$HOME/adastra_warpx.profile``, and load it into your shell after a login:
+
+.. code-block:: bash
+
+   source $HOME/adastra_warpx.profile
+
+And since Adastra does not yet provide a module for them, install c-blosc and ADIOS2:
+
+.. code-block:: bash
+
+   export CMAKE_PREFIX_PATH=${HOME}/sw/adastra/gpu/c-blosc-1.21.1:$CMAKE_PREFIX_PATH
```

Comment:
Is the default home quota large enough? I started transitioning on NERSC (10GB default) now to other locations.

Here, you could use `$WORKDIR/sw` or `$SHAREDWORKDIR/$USER/sw` it seems.

Reply:
Ping @lucafedeli88

Reply:
ping @NeilZaim @lucafedeli88 

can be a follow-up :)